src/cvv-field.tsx

```diff
@@ -95,6 +98,8 @@ const CvvField: React.FC<CvvFieldProps> = ({
         onChangeValue,
     });
 
+    const iconSize = pxToRem(16);
```

Comment:
Icon size was smaller than in other input fields. Double check if this makes sense @yceballost

Reply:
it is expected